secrets/dataplane-observability-config.yaml.sample

```diff
@@ -0,0 +1,40 @@
+---
+# [required]
+remote_write_configuration:
+  # [required] V2 URL endpoint of the Prometheus instance to be used by the data planes
+  # to send metrics. The Prometheus instance has to offer a Remote Write endpoint.
+  # If oidc_configuration is set that endpoint has to be offered under the
+  # /api/v1/metrics/<tenant>/api/v1/receive" URL path, where <tenant> is defined by the
+  # --observability-red-hat-sso-tenant Fleet Manager flag.
+  # If oidc_coniguration is set that endpoint has to be offered under the
+  # "/" URL path.
+  # The format of remote_write_url is `<url-scheme>:<hostname>:<port>` where
+  # <port> is optional.
+  remote_write_url: # fill here
+  # [optional] OIDC configuration used to retrieve OIDC tokens to authenticate against the
+  # Prometheus Remote Write endpoint URL. If not provided, the
+  # KAS Fleet Manager - Data Plane cross communication OIDC token will be used.
```

Comment:
`KAS Fleet Manager - Data Plane cross communication OIDC token`. What's this? 🤔 Is this talking about our current oidc config and credentials we use with the data plane token refreshers?

Reply:
I can't think of anything else apart from what is mentioned here at the top of my head 🤔 thanks for clarifying!